sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationTokenCredential.java

```diff
@@ -35,13 +35,13 @@
     private boolean isClosed = false;
 
     /**
-     * Create with serialized JWT token
-     * 
-     * @param token serialized JWT token
+     * Create with serialized JWT initialToken
+     *
+     * @param initialToken serialized JWT initialToken
      */
-    public CommunicationTokenCredential(String token) {
-        Objects.requireNonNull(token, "'token' cannot be null.");
-        setToken(token);
+    public CommunicationTokenCredential(String initialToken) {
```

Comment:
As I remember, we had talked and decided to use `token` and not `initialToken` as the name, so please revert the change to make it match C#. Sorry ...

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Common/src/CommunicationTokenCredential.cs